dgraph/cmd/alpha/run.go

```diff
@@ -712,8 +717,6 @@ func run() {
 	setupCustomTokenizers()
 	x.Init()
 	x.Config.PortOffset = Alpha.Conf.GetInt("port_offset")
-	x.Config.Limit = z.NewSuperFlag(Alpha.Conf.GetString("limit")).MergeAndCheckDefault(
-		worker.LimitDefaults)
 	x.Config.LimitMutationsNquad = int(x.Config.Limit.GetInt64("mutations-nquad"))
 	x.Config.LimitQueryEdge = x.Config.Limit.GetUint64("query-edge")
```

Comment:
just a quick thought wouldnt this be great to actually have struct say 
`type Limit struct {
 ...
...
}`

I am sensing with super flags we are just putting too much reliance over x.SuperFlag.Get* function and causing too much noise in various package configuration. Like limit is super flag but x.Config has all the individual limits options. Seems strange to me and bit unstructured